### PR TITLE
feat(eventbridge-monitor): making an event bridge monitor

### PR DESCRIPTION
**Pull request by bassrock**
## Goal

Setup a lambda that simply logs all events that have a `detail-type` field that appear in our Pocket Eventbridge.

The idea is this can help with debugging of event based systems as long as the event occurred in the last 14 days.

## Future

In the future we could make the detail-type & source data log to Grafana, Cloudwatch, or Snowplow so that we can check in on event volumes and patterns.  I chose to leave that out for now.

## Todo

- [x] setup sentry dsn

Screenshot 2024-03-07 at 8 36 21 AM

### CODE REVIEW

File: lambdas/account-data-deleter-batch-delete/src/index.spec.ts [hunk 1]
```diff
@@ -22,25 +22,44 @@ describe('deleteUsers spec test', () => {
     jest.clearAllMocks();
   });
 
-  afterAll(() =>  {
+  afterAll(() => {
```
Comment (bassrock) [inline lambdas/account-data-deleter-batch-delete/src/index.spec.ts:25]: lint fixes..

File: lambdas/pocket-event-bridge-monitor/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  dsn: config.sentry.dsn,
+  release: config.sentry.release,
+  environment: config.app.environment,
+  serverName: config.app.name,
+});
+
+export async function handlerFn(event: SQSEvent): Promise<any> {
+  for await (const record of event.Records) {
+    try {
+      const message = JSON.parse(JSON.parse(record.body).Message);
```
Comment (bassrock) [inline lambdas/pocket-event-bridge-monitor/src/index.ts:16]: All that code for this!

File: lambdas/pocket-event-bridge-monitor/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      serverLogger.info('Event received', message);
```
Comment (bassrock) [inline lambdas/pocket-event-bridge-monitor/src/index.ts:17]: using serverLogger, for timestamped structured logs.

File: infrastructure/pocket-event-bridge/src/event-rules/all-events/allEventRules.ts [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+   * @private
+   */
+  private createAllEventRules(logGroup: CloudwatchLogGroup) {
+    const allEventRuleProps: PocketEventBridgeProps = {
+      eventRule: {
+        name: `${config.prefix}-${eventConfig.name}-Rule`,
+        eventPattern: {
+          'detail-type': [
+            {
+              exists: true,
```
Comment (kschelonka) [inline infrastructure/pocket-event-bridge/src/event-rules/all-events/allEventRules.ts:63]: sweet, that's easy

**Review by kschelonka — APPROVED** [review]
slick

**Pull request merged**